### DISCUSSION TITLE
Button Alignment In Form Fields

### DISCUSSION THREAD

**Discussion by hajanajubudeen88** [fomantic/Fomantic-UI#651] [Collections (form, grid, menu, message, table, ...)] [answered]
# Help Wanted
Hi Team,
I need the button in same row with using FUI class name, someone please guide me how to fix this issue 
## Problem
Here is my fiddle link ( https://jsfiddle.net/hajanajubudeen88/y4967fvn/ )

1. Here i create a form with 4 cols 
2. In that i have 3 text fields and one button
3. The button is not in proper alignment 

please some help to fix with FUI Class name

**Comment by lubber-de** [comment, marked as answer]
## Two solutions here:
#### Add a label using whitespace to force the same height
https://jsfiddle.net/71ozhqre/1/

#### Fix it via CSS as done similiar in #613 by @GammaGames.
https://jsfiddle.net/71ozhqre

**Comment by hajanajubudeen88** [comment]
Thank You
